docs/demos/easy_classification.ipynb

```diff
@@ -69,3579 +69,9 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    Y = np.round(1. / (1+np.exp(-Y)))
</pre><p>Small issue with the simulation, this sets Y to the most likely class given X &amp; beta, but I believe we should be sampling Y from P(Y = 1) = 1 / (1 + np.exp(-Y_{line_10})). Check out the relationship between true and pred betas below, I think the direct assignment of Y to the most likely class is causing some major deviation between the true and pred.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Looks like this is already online and not specific to this PR. I've made a separate issue #169 to address this.

Reply:
PR filed to fix this: #170